sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -322,7 +321,7 @@ def _rows_to_raw_statement_events(self, rows):
                 "ddagentversion": datadog_agent.get_version(),
                 "ddsource": "sqlserver",
                 "dbm_type": "rqt",
-                "ddtags": ",".join(self.tags),
+                "ddtags": ",".join(self._check.tag_manager.get_tags()),
```

Comment:
Why does this use `",".join()` but all other events just pass the tag list? I know it's how it was previously but curious if it matters